tfhe-benchmark/benches/high_level_api/erc20.rs

```diff
@@ -689,7 +771,7 @@ fn main() {
             bench_name,
             "FheUint64",
             "transfer::whitepaper",
-            par_transfer_whitepaper::<FheUint64>,
+            transfer_whitepaper::<FheUint64>,
         );
         cuda_bench_transfer_throughput(
```

Comment:
I think for all the other cuda throughput cases you need to make the change as well, to use the non-parallel variant.

Reply:
Aaah ok makes sense! :pray: